Source/ThirdParty/libwebrtc/Source/webrtc/modules/rtp_rtcp/source/rtp_format_h265.cc

```diff
@@ -201,7 +204,9 @@ bool RtpPacketizerH265::PacketizeSingleNalu(size_t fragment_index) {
                       << limits_.max_payload_len;
     return false;
   }
-  RTC_CHECK_GT(fragment.size(), 0u);
+  if (fragment.size() == 0u) {
+    return false;
+  }
   packets_.push(PacketUnit(fragment, true /* first */, true /* last */,
```

Comment:
@ddkilzer  Thanks for the catch. That rarely happens to my scenario but looks good to me.

Reply:
Thanks @xingri.  Are you working on merging this code upstream?  If so, is there anything I can do to make that job easier?